### PR TITLE
Version unifying

### PR DESCRIPTION
**Pull request by rafalste**
Signed-off-by: Rafal Stefanowski <rafal.stefanowski@intel.com>

### CODE REVIEW

File: utils/cas_version_gen [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        error "couldn't create version file"
+    fi
+    echo "CAS_VERSION_MAIN=$CAS_VERSION_MAIN" > "$VERSION_FILE"
+    echo "CAS_VERSION_MAJOR=$CAS_VERSION_MAJOR" >> "$VERSION_FILE"
+    echo "CAS_VERSION_MINOR=$CAS_VERSION_MINOR" >> "$VERSION_FILE"
+    echo "CAS_VERSION_BUILD=$CAS_VERSION_BUILD" >> "$VERSION_FILE"
+    echo "CAS_VERSION_RELEASE=$CAS_VERSION_RELEASE" >> "$VERSION_FILE"
+    echo "CAS_VERSION=$CAS_VERSION" >> "$VERSION_FILE"
+    echo "LAST_COMMIT_HASH=$LAST_COMMIT_HASH" >> "$VERSION_FILE"
+    echo "LAST_COMMIT_HASH_ABBR=$LAST_COMMIT_HASH_ABBR" >> "$VERSION_FILE"
```
Comment (imjfckm) [inline utils/cas_version_gen:72]: Do we need both abbreviated and full hash? I think abbreviation is constructed in such way that it is unique across repo.

Comment (rafalste) [reply utils/cas_version_gen:72]: I've added full hash for precision and abbreviated for convenience. Let me know if I should remove one of them.

Comment (rafalste) [reply utils/cas_version_gen:72]: And also I don't know if the abbreviation is globally unique.

Comment (imjfckm) [reply utils/cas_version_gen:72]: Well, TBH, I don't think it matters that much, leave it ;)

**Pull request merged**